clients/client-testextension/src/main/java/org/projectnessie/client/ext/NessieClientFactory.java

```diff
@@ -30,10 +30,12 @@ public interface NessieClientFactory {
   NessieApiVersion apiVersion();
 
   @Nonnull
+  @jakarta.annotation.Nonnull
```

Comment:
Do we need `jakarta` annotations on the client side? Quarkus does not "process" the java client, does it?

Is this about API parameter validation? If so, would be only have to hava `jakarta` annotations on interface boundaries?